### PR TITLE
Release/1.2.46

### PR DESCRIPTION
**Pull request by ttypic**
## Summary by CodeRabbit

- **New Features**
	- Added new experimental fields to the Message class: `action`, `serial`, and `createdAt`.
  
- **Documentation**
	- Updated the changelog to reflect the new version `1.2.46`.
	- Revised the `CONTRIBUTING.md` file for clearer development and testing instructions.
	- Updated dependency versions in the `README.md` to `1.2.46`.

- **Chores**
	- Incremented the `VERSION_NAME` in `gradle.properties` to `1.2.46`.
	- Updated test parameters to reflect the new version in tests.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,18 @@
 # Change Log
 
+## [1.2.46](https://github.com/ably/ably-java/tree/v1.2.46)
+
+[Full Changelog](https://github.com/ably/ably-java/compare/v1.2.45...v1.2.46)
+
+**Implemented enhancements:**
+
+- New `action`, `serial`, `createdAt` `Message` fields has been added
```
Comment (AndyTWF) [inline CHANGELOG.md:9]:
```suggestion
- New `action`, `serial`, `createdAt` `Message` fields have been added.
```

Comment (AndyTWF) [inline CHANGELOG.md:9]:
```suggestion
- New `action`, `serial`, `createdAt` `Message` fields have been added.
```

Comment (AndyTWF) [reply CHANGELOG.md:9]: I think we also need to add a notice that these fields are new/experimental/not-stable etc.

**Review by sacOO7 — APPROVED** [review]
LGTM, seems https://github.com/ably/ably-java/pull/1049#discussion_r1861970992 can be addressed before merging

**Review by AndyTWF — APPROVED** [review]

**Pull request merged**